app/controllers/collection_items_controller.rb

```diff
@@ -81,6 +84,16 @@ def load_collectible_item
     end
   end
 
+  def load_user
+    unless @collection
+      @user = User.current_user
+    end
+  end
+
+  def add
+    # TODO: Find some way to let mods add works from inside the collection
```

Comment:
I'm always nervous about leaving TODOs in the code, as they tend never to get done - is this something that is likely to be addressed in the near future? If not, can the `add` method be removed or if it needs to exist, is there some placeholder behaviour that could be created for it? (not that the latter is much better than a TODO, so this is just some suggestions)